docs/dev/table/connectors/index.md

```diff
@@ -0,0 +1,268 @@
+
+* This will be replaced by the TOC
+{:toc}
+
+Supported Connectors
+------------
+
+Flink natively support various connectors. The following tables list all available connectors.
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left">Name</th>
+        <th class="text-center">Version</th>
+        <th class="text-center">Source</th>
+        <th class="text-center">Sink</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>Filesystem</td>
+      <td></td>
+      <td>Bounded and Unbounded Scan, Lookup</td>
+      <td>Streaming Sink, Batch Sink</td>
+    </tr>
+    <tr>
+      <td>Elasticsearch</td>
+      <td>6.x & 7.x</td>
+      <td>Not supported</td>
+      <td>Streaming Sink, Batch Sink</td>
+    </tr>
+    <tr>
+      <td>Apache Kafka</td>
+      <td>0.10+</td>
+      <td>Unbounded Scan</td>
+      <td>Streaming Sink, Batch Sink</td>
+    </tr>
+    <tr>
+      <td>JDBC</td>
+      <td></td>
+      <td>Bounded Scan, Lookup</td>
+      <td>Streaming Sink, Batch Sink</td>
+    </tr>
+    <tr>
+      <td><a href="{{ site.baseurl }}/dev/table/connectors/hbase.html">Apache HBase</a></td>
+      <td>1.4.x</td>
+      <td>Bounded Scan, Lookup</td>
+      <td>Streaming Sink, Batch Sink</td>
+    </tr>
+    </tbody>
+</table>
+
+{% top %}
+
+How to use connectors
+--------
+
+Flink supports to use SQL CREATE TABLE statement to register a table. One can define the name of the table, the schema of the table, the connector options for connecting to an external system.
+
+The following code shows a full example of how to connect to Kafka for reading Json records.
+
+<div class="codetabs" markdown="1">
+<div data-lang="SQL" markdown="1">
+{% highlight sql %}
+CREATE TABLE MyUserTable (
+  -- declare the schema of the table
+  `user` BIGINT,
+  message STRING,
+  ts TIMESTAMP,
+  proctime AS PROCTIME(), -- use computed column to define proctime attribute
+  WATERMARK FOR ts AS ts - INTERVAL '5' SECOND  -- use WATERMARK statement to define rowtime attribute
+) WITH (
+  -- declare the external system to connect to
+  'connector' = 'kafka',
+  'topic' = 'topic_name',
+  'scan.startup.mode' = 'earliest-offset',
+  'properties.bootstrap.servers' = 'localhost:9092',
+  'format' = 'json'   -- declare a format for this system
+)
+{% endhighlight %}
+</div>
+</div>
+
+In this ways the desired connection properties are converted into normalized, string-based key-value pairs. So-called [table factories](sourceSinks.html#define-a-tablefactory) create configured table sources, table sinks, and corresponding formats from the key-value pairs. All table factories that can be found via Java's [Service Provider Interfaces (SPI)](https://docs.oracle.com/javase/tutorial/sound/SPI-intro.html) are taken into account when searching for exactly-one matching table factory.
+
```

Comment:
What does the `normalized` mean ?

Reply:
I copied this paragraph from original connector page. I think it means to lower case?

Reply:
`In this ways` -> `In these ways` or `In this way`

Reply:
I removed `normalized`, because it is about implementation. Users shouldn't care about this.